apps/docs/content/components/PricingOptions.mdx

```diff
@@ -0,0 +1,747 @@
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/buy">
+      Buy now
+    </PricingOptions.PrimaryAction>
+    <PricingOptions.SecondaryAction href="/contact">
+      Contact sales
+    </PricingOptions.SecondaryAction>
+  </PricingOptions.Item>
+
+  <PricingOptions.Item>
+    <PricingOptions.Label>Recommended</PricingOptions.Label>
+    <PricingOptions.Heading>Copilot Business</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Copilot personalized to your organization throughout the software
+      development lifecycle. Requires GitHub Enterprise Cloud.
+    </PricingOptions.Description>
+    <PricingOptions.Price currencySymbol="$" trailingText="per user / month">
+      39
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListItem>
+        Everything in Copilot Business plus:
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/buy">
+      Join waitlist
+    </PricingOptions.PrimaryAction>
+  </PricingOptions.Item>
+</PricingOptions>
+```
+
+### Three items
+
+This is the three-item layout for the pricing options component.
+
+```jsx live
+<PricingOptions>
+  <PricingOptions.Item>
+    <PricingOptions.Heading>Copilot Individual</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Code completions, Chat, and more for indie developers and freelancers.
+    </PricingOptions.Description>
+    <PricingOptions.Price
+      currencySymbol="$"
+      trailingText="per month / $100 per year"
+    >
+      10
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Security vulnerability filter
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.Footnote>
+      Free for verified students, teachers, and maintainers of popular open
+      source projects.
+    </PricingOptions.Footnote>
+    <PricingOptions.PrimaryAction href="/buy">
+      Start a free trial
```

Comment:
Probably just a docs issue, but this is wrapping on two lines..

<img width="238" alt="Screenshot 2024-03-20 at 11 45 27" src="https://github.com/primer/brand/assets/13340707/9fff8944-be66-4ff7-a6d3-d9d11bf1c2fa">